keps/NNNN-kep-template/production-readiness.md

```diff
@@ -0,0 +1,139 @@
+# Production Readiness Review Questionnaire
```

Comment:
Given you move it here, we probably should remove it from the community repo, to avoid maintaining the same thing in two different places.

Reply:
Yes, that's the plan. Different repo though so it will be a separate PR.